ReactiveCocoa/CocoaTarget.swift

```diff
@@ -3,14 +3,29 @@ import ReactiveSwift
 import enum Result.NoError
 
 /// A target that accepts action messages.
-internal final class CocoaTarget: NSObject {
-	let observer: Observer<(), NoError>
-
-	init(_ observer: Observer<(), NoError>) {
+internal final class CocoaTarget<Value>: NSObject {
+	private let observer: Observer<Value, NoError>
+	private let transform: (Any?) -> Value
+	
+	internal init(_ observer: Observer<Value, NoError>, transform: @escaping (Any?) -> Value) {
 		self.observer = observer
+		self.transform = transform
```

Comment:
So this was made to share the implementation by letting the user decide how the `Any?` sender is transformed to feed the observer. Existing users of `CocoaTarget` need not pass through the sender, so it was `Any? -> Void`, while in this case it is `Any? -> UIGestureRecognizer`.